presto-example-http/pom.xml

```diff
@@ -26,13 +26,6 @@
             <artifactId>json</artifactId>
         </dependency>
 
-        <dependency>
-            <!-- airlift json has a dependency on joda -->
-            <groupId>io.airlift</groupId>
-            <artifactId>joda-to-java-time-bridge</artifactId>
-            <scope>runtime</scope>
```

Comment:
Is this the only one that does not need the bridge anymore?

Reply:
I checked before and all of the other plugins that use it have a direct dependency on `joda-time`. @haozhun helpfully put it directly after the `joda-time` dependency in all the POMs, so it was easy to verify using `ack -B10 joda-to-java`.